functionary/ui/templatetags/social_helper.py

```diff
@@ -21,3 +25,16 @@ def find_account(context, provider_id: str) -> ProviderAccount | None:
             if account.provider == provider_id:
                 return account.get_provider_account().account
     return None
+
+
+@register.simple_tag()
+def configured_providers() -> list[Tuple[str, Provider]] | None:
+    """This tag returns a list of tuples of the configured Providers.
+
+    Returns:
+        List of tuples of configured provider name and Provider
+    """
+    return [
+        (app.name, providers.registry.by_id(app.provider))
```

Comment:
If you made this just a dictionary, would you be able to avoid using the `provider.0/1.*` in the templates?